covfee/client/tasks/continuous_annotation/index.tsx

```diff
@@ -238,6 +238,7 @@ const ContinuousAnnotationTask: React.FC<Props> = (props) => {
   // We get a reference to the VideoJS player and assign event
   // listeners to it.
   const videoPlayerRef = useRef<VideoJsPlayer>(null)
+  const [, setIsVideoPlayerReady] = useState(false)
```

Comment:
Is it okay to nos specify the variable?

Reply:
Yes. I don't really needed to "read" the variable. I just need to set the state. Copilot this is the way to do it in Javascript to ignore a non needed variable.